text/0064-store-file-system-permissions-in-metadata.md

```diff
@@ -0,0 +1,56 @@
+- FIP: 64
+- title: Store file-system permissions in file and dir matadata
+- author: Sabyasachi Patra, @asabya
+- status: draft
+- created: 2023-01-17
+
+# Summary
+File system permission is a set of rules that determine who can access and perform certain actions on a file or directory in a computer's file system. These permissions can be applied to users and groups, and can include the ability to read, write, execute, or delete files.
```

Comment:
So I guess then FDA and any other client that mounts personal storage will handle the user/group binding dynamically. this has default like `user:user` who runs the FDA app right? and optionally it could be changed trough setting.

Reply:
yes. 

The logged-in user at the OS level will be treated as owner of the file.